### DISCUSSION TITLE
Alias Autocomplete does not work if blank lines exist between clauses

### DISCUSSION THREAD

**Discussion by Stephonomon** [dbeaver/dbeaver#14319] [Q&A]
-->

#### System information: 
-Windows 10 Enterprise Version 1909
- DBeaver version: 21.1.3
- Additional extensions: None

#### Connection specification:
- Database name and version: Netezza
- Driver name: C:\Program Files\DBeaver\drivers\netezza\nzjdbc3.jar
- Do you use tunnels or proxies (SSH, SOCKS, etc)? No

#### Describe the problem you're observing:
When trying to get a column autocomplete on a table alias, I get a "no proposals" message under certain conditions. If there is any blank lines between my Clauses, it does not autocomplete, but if there are no blank lines, autocomplete works.

#### Steps to reproduce, if exist:
**Working Example**
```SELECT 
dt.
FROM 
admin.dim_date as dt
```
![image](https://user-images.githubusercontent.com/9829881/138122420-e6b02468-8d72-40b5-b481-740a5b2c0b40.png)

**Non-working example**

```
SELECT 
dt.

FROM 
admin.dim_date as dt
```
![image](https://user-images.githubusercontent.com/9829881/138122288-2c22f6d5-8427-4c46-8966-78a4c45e3605.png)

**Comment by LonwoLonwo** [comment]
Hello @Stephonomon
Thanks for the report.
This is expected behavior for SQL Editor when working with blank lines inside the query.
You can change it in Preferences->Editors->SQL Editor->SQL Processing
`Blank line is statement delimiter` must be disabled in your case.

![2021-10-21 00_58_04-DBeaver Ultimate 21 2 3 - _postgres_ Script-10](https://user-images.githubusercontent.com/45152336/138178755-b3261a4a-bb4d-40d1-8d64-df0019225abb.png)

![2021-10-21 00_59_03-](https://user-images.githubusercontent.com/45152336/138178779-56a1a316-4e32-4572-81aa-f71b013ecb2e.png)

**Reply by Ackbach** [reply]
I would advise changing the shipping default so that the box is unchecked. I, for one, strongly prefer blank lines in my SQL to make it more readable, and unchecking the box allows Autocomplete to still work, which is super-nice. At the very least, this behavior needs to be documented better. I had a colleague search for hours trying to find out why this was happening.
